server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -118,14 +82,24 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
             return;
         }
 
-        String youtubeResBody = getYoutubeRes(API_KEY, accessToken.toString());
-        int numVideos = Integer.parseInt(req.getParameter("num_videos"));
+        String numVideosParam = req.getParameter("num_videos");
+        int numVideos = DEFAULT_NUM_VIDS;
+        if (numVideosParam != null) {
+            numVideos = Integer.parseInt(numVideosParam);
+        }
+
+        String youtubeResBody = getYoutubeRes(API_KEY, accessToken.toString(), numVideos);
+        JsonObject youtubeJsonObj = JsonParser.parseString(youtubeResBody).getAsJsonObject();
+
+        List<VideoGenreCount> genreCountList = new ArrayList<>();
```

Comment:
Delete this. Let `YoutubeGenres` declare its own variable instead of passing it an otherwise unused variable.